From 4efaf6659a37a4860f67c9131e04cbda12210658 Mon Sep 17 00:00:00 2001
From: Dominggoes Isakh <drjisakh@gmail.com>
Date: Tue, 9 Sep 2025 13:54:34 -0400
Subject: [PATCH 2/5] gts2-common: Allow to access bootable/deprecated-ota
 targets

gts2-common is still using non-AB update.
Non-AB code will be moved to a separated namespace, add
deprecated-ota to gts2-common namespace list ahead of time to avoid
breakage.

Test: th
Bug: 324360816 | AOSP
Change-Id: I14f4893b3404a2ceb1e6b810f4bfdc1cf351ccde
---
 device-common.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/device-common.mk b/device-common.mk
index f4fa4d4..069f814 100644
--- a/device-common.mk
+++ b/device-common.mk
@@ -278,7 +278,10 @@ PRODUCT_PACKAGES += \
 # Setup dalvik vm configs
 $(call inherit-product, frameworks/native/build/phone-xhdpi-4096-dalvik-heap.mk)
 
+# Update
 AB_OTA_UPDATER := false
+PRODUCT_SOONG_NAMESPACES += \
+    bootable/deprecated-ota
 
 # Soong namespaces
 PRODUCT_SOONG_NAMESPACES += \
-- 
2.45.2

